packages/beacon-node/src/util/blobs.ts

```diff
@@ -46,3 +63,44 @@ export function computeBlobSidecars(
     return {index, blob, kzgCommitment, kzgProof, signedBlockHeader, kzgCommitmentInclusionProof};
   });
 }
+
+/**
+ * Turns a SignedBeaconBlock and an array of Blobs from a given slot into an array of
+ * DataColumnSidecars that are ready to be served by gossip and req/resp.
+ *
+ * Implementation of get_data_column_sidecars
+ * https://github.com/ethereum/consensus-specs/blob/dev/specs/_features/eip7594/das-core.md#get_data_column_sidecars
+ */
+export function computeDataColumnSidecars(
```

Comment:
don't we already have this implemented  @matthewkeil ? or is this an extraction from the branch into the main PRs you were talking about?